samples/highcharts/dragdrop/undraggable-points/demo.js

```diff
@@ -0,0 +1,62 @@
+Highcharts.chart('container', {
+    yAxis: {
+        softMin: -200,
+        softMax: 200
+    },
+
+    xAxis: {
+        min: 0.5,
+        max: 3.5
+    },
+
+    tooltip: {
+        enabled: false
+    },
+
+    title: {
+        text: 'Drag & drop disabled for specific points.'
+    },
+
+    series: [{
+        dragDrop: {
+            draggableY: true
+        },
+
+        point: {
+            events: {
+                drag: function () {
+                    if (this.x === 2) {
+                        return false;
+                    }
+                },
+                drop: function () {
+                    if (this.x === 2) {
+                        return false;
+                    }
+                }
+            }
+        },
+
+        dataLabels: {
+            enabled: true,
+            y: -6,
+            formatter: function () {
+                return this.x === 2 ? 'undraggable' : '';
+            }
```

Comment:
Prefer using `format` instead. To limit the data label to only the undraggable point, it can be defined directly on the `data` item. Alternatively use [templating](https://www.highcharts.com/docs/chart-concepts/templating).